src/Microsoft.AspNetCore.DataProtection/EphemeralDataProtectionProvider.cs

```diff
@@ -48,10 +48,10 @@ public EphemeralDataProtectionProvider(IServiceProvider services)
                 keyringProvider = new EphemeralKeyRing<ManagedAuthenticatedEncryptionSettings>();
             }
 
-            var logger = services.GetLogger<EphemeralDataProtectionProvider>();
+            var logger = loggerFactory?.CreateLogger<EphemeralDataProtectionProvider>();
             logger?.UsingEphemeralDataProtectionProvider();
```

Comment:
This is the kind of thing that I was talking about when we discussed in person earlier - this logging statement doesn't include any detail about the guts of this class, it just states what provider we're using. That logging could come from anywhere